package.json

```diff
@@ -8,7 +8,7 @@
   "license": "GPL-3.0",
   "private": false,
   "scripts": {
-    "start": "webpack-dev-server --mode development",
+    "start": "webpack-dev-server --hot --mode development",
```

Comment:
🔥

Reply:
👍

Reply:
definitely recommend either split-screen or dual screening with this feature. it's a dream compared to hard-reloading the page every time.